analyzers/src/SonarAnalyzer.CSharp/Rules/MemberInitializerRedundant.cs

```diff
@@ -276,6 +338,9 @@ protected override bool IsBlockInvalid(Block block)
                                 }
                             }
                             break;
+                        default:
+                            // continue search
+                            break;
```

Comment:
Line 364: Name can be simplified.

Reply:
could you please detail? in this CR line 364 is `return true;`